example-custom-format.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const http = require('http')
+const server = http.createServer(handle)
+
+const logger = require('./')({
+  quietReqLogger: true,
+  transport: {
+    target: 'pino-http-print',
+    options: {
+      destination: 1,
+      all: true,
+      colorize: false,
+      translateTime: true
+    }
+  }
+})
+
+function handle (req, res) {
+  logger(req, res)
+  res.end('hello world')
+}
+
+server.listen(3000)
```

Comment:
It's better if you use standard error to print something out after the listen is complete and then use that as a trigger to send the incoming http message.